Packs/DigitalShadows/Integrations/DigitalShadows/DigitalShadows.yml

```diff
@@ -1861,7 +1861,7 @@ script:
     description: Batch retrieve specic tags by their ids
   isfetch: true
 fromversion: 5.0.0
-defaultmapperin: Digital Shadows Incident Mapper
-defaultclassifier: Digital Shadows Incident Classifier
+defaultmapperin: DigitalShadowsIncidentMapper_V2
```

Comment:
Didn't find those values in the repo - maybe you wanted to refer the new ones located in this PR?

Reply:
ops will need to update those